homeassistant/components/tradfri/cover.py

```diff
@@ -59,11 +60,20 @@ def current_cover_position(self):
         """Close cover."""
         await self._api(self._device_control.set_state(100))
 
+    async def async_stop_cover(self, **kwargs):
+        """Close cover."""
+        await self._api(self._device_control.trigger_blind())
+
     @property
     def is_closed(self):
         """Return if the cover is closed or not."""
         return self.current_cover_position == 0
 
+    @property
+    def device_state_attributes(self):
+        """Return the devices' state attributes."""
+        return {ATTR_BATTERY: self._device.device_info.battery_level}
```

Comment:
If I understood the conversation with @MartinHjelmare correctly, this is not the preferred way of doing it. Battery properties of a device should be represented as battery devices.

Reply:
Correct, the issue is that If this is added as a separate entity for the battery sensor, it gets a separate observer that observes the same device as the blind itself.

This again causes stability issues with multiple tradfri platforms.
Users who have tested that has reported unresponsive blinds and lights.

I can remove the battery part of the PR and just add stop to the blinds, but the battery is an important metric for many users.

Reply:
Let's remove the battery attribute for now.